### PR TITLE
Event calendar: Respect voucher for availability

### PR DESCRIPTION
**Pull request by raphaelm**

### CODE REVIEW

File: src/pretix/base/models/event.py [hunk 1]
```diff
@@ -1454,10 +1467,10 @@ def _seats(self, ignore_voucher=None):
         return qs_annotated
 
     @classmethod
-    def annotated(cls, qs, channel='web'):
+    def annotated(cls, qs, channel='web', voucher=None):
         from .items import SubEventItem, SubEventItemVariation
 
-        qs = super().annotated(qs, channel)
+        qs = super().annotated(qs, channel, voucher=voucher)
```
Comment (wiffbi) [inline src/pretix/base/models/event.py:1473]: It works named, but to make it more consistent:
```suggestion
        qs = super().annotated(qs, channel, voucher)
```

**Review by wiffbi — APPROVED** [review]
Looks good, just a minor comment regarding consistency.

Should we respect the voucher when showing the subevents-overview in the widget?

**Comment by raphaelm** [comment]
The widget currently does not allow passing a voucher to an event list according to it's documentation, so I'd hold off for now

File: src/pretix/base/models/event.py (hunk 1 above)
Comment (raphaelm) [reply src/pretix/base/models/event.py:1473]: Yeah, but using more keyword arguments generally makes the code easier to maintain in the future

**Pull request merged**